mkdocs/__main__.py

```diff
@@ -99,9 +100,10 @@ def cli():
 @click.option('-a', '--dev-addr', help=dev_addr_help, metavar='<IP:PORT>')
 @click.option('-s', '--strict', is_flag=True, help=strict_help)
 @click.option('-t', '--theme', type=click.Choice(theme_choices), help=theme_help)
+@click.option('-e', '--theme-dir', type=click.Path(), help=theme_dir_help)
```

Comment:
`-d` is used for [`--site_dir`][1] on the build command. And as `--theme_dir` is being added to both `build` and `serve`, the short flag should be consistent (`--site_dir` makes no sense for `serve`).

[1]: https://github.com/mkdocs/mkdocs/blob/master/mkdocs/__main__.py#L127